lldb/source/Symbol/Type.cpp

```diff
@@ -1082,6 +1082,19 @@ bool TypeImpl::GetDescription(lldb_private::Stream &strm,
   return true;
 }
 
+CompilerType TypeImpl::FindNestedType(ConstString name) {
+  auto type_system = GetTypeSystem(false);
```

Comment:
Can you add an inline comment for this false? It's not obvious what it means here. Something like:
```
auto type_system = GetTypeSystem(/*param_name*/ false);
```

Reply:
Done!